all/build.gradle

```diff
@@ -3,21 +3,23 @@ description = "gRPC: All"
 // Make sure that no transitive dependencies are included.
 configurations.compile.transitive = false
 
-for (subproject in rootProject.subprojects) {
-    if (subproject == project) {
-        continue
-    }
+def subprojects = [
+  project(':grpc-auth'),
+  project(':grpc-core'),
+  project(':grpc-netty'),
+  project(':grpc-okhttp'),
+  project(':grpc-protobuf'),
+  project(':grpc-protobuf-nano'),
+  project(':grpc-stub'),
+]
+
+for (subproject in subprojects) {
     evaluationDependsOn(subproject.path)
 }
+evaluationDependsOn(project(':grpc-interop-testing').path)
```

Comment:
Actually, how do you feel about the `evaluationDependsOn` still looping over all of `rootProject.subprojects`, but doing `project(':grpc-interop-testing')` like it is now in the `jacocoMerge`?